src/web/client/utilities/fileAndEntityUtil.ts

```diff
@@ -89,6 +90,18 @@ export function getFileName(fsPath: string) {
     return fsPath.split(/[\\/]/).pop();
 }
 
+export function getDefaultLanguageCodeWeb() {
```

Comment:
you should not be fetching languages on the fly in copilot using these functions - but as part of init call - check the way we are passing orgId, envId etc. That is the right pattern to use. Otherwise you will land in cross module dependency.